api/v1beta1/keystoneapi_types.go

```diff
@@ -175,7 +175,7 @@ type KeystoneAPIStatus struct {
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
 //+kubebuilder:printcolumn:name="Status",type="string",JSONPath=".status.conditions[0].status",description="Status"
-//+kubebuilder:printcolumn:name="Reason",type="string",JSONPath=".status.conditions[0].reason",description="Reason"
```

Comment:
I see that the `KeystoneService` CRD still has `Status`, `Reason` and `Message` in its `printcolumns`.  Did you mean to remove `Reason` here from `KeystoneAPI` and not follow the same pattern?

Reply:
I have not finished the keystoneservice yet. wanted to submit this draft to get early feedback. will update it now

Reply:
changed to be in sync, I'll submit a new PR or at least a new commit for other changes to the keystoneservice controller